SavedInstances/SavedInstances.lua

```diff
@@ -2893,28 +2930,54 @@ function core:RefreshDailyWorldQuestInfo()
     local title = GetQuestLogTitle(GetQuestLogIndexByID(BountyInfo.questID))
     local timeleft = C_TaskQuest.GetQuestTimeLeftMinutes(BountyInfo.questID)
     local _, _, isFinish, questDone, questNeed = GetQuestObjectiveInfo(BountyInfo.questID, 1, false)
-    if not isFinish and timeleft then
+    if timeleft then
       if timeleft > 2880 then
-        t.DailyWorldQuest.days2 = {}
+        if t.DailyWorldQuest.days2 then else t.DailyWorldQuest.days2 = {} end
```

Comment:
Eh, I just saw oscarub did this all over the place and you just matched his style, nvm then.